nixos/tests/xscreensaver.nix

```diff
@@ -0,0 +1,64 @@
+import ./make-test-python.nix ({ pkgs, lib, ... }: {
```

Comment:
Tests should be in a separated commit

Reply:
Done! Also split out the package and the module while I was at it.